fury/actors/peak.py

```diff
@@ -0,0 +1,343 @@
+
+        Returns
+        -------
+        color_array : vtkDataArray
+            vtk scalar array with name 'colors'.
+        colors_are_scalars : bool
+            indicates whether or not the colors are scalars to be interpreted
+            by a colormap.
+        global_opacity : float
+            returns 1 if the colors array doesn't contain opacity otherwise -1.
+
+        """
+        pnts_per_line = 2
+        num_pnts = len(points)
+        num_lines = num_pnts // pnts_per_line
+        colors_are_scalars = False
+        global_opacity = 1
+        if colors is None or colors == 'rgb_standard':
+            # Automatic RGB colors
+            colors = np.asarray((0, 0, 0))
+            color_array = numpy_to_vtk_colors(
+                np.tile(255 * colors, (num_pnts, 1)))
+        elif type(colors) is tuple:
+            global_opacity = 1 if len(colors) == 3 else -1
+            colors = np.asarray(colors)
+            color_array = numpy_to_vtk_colors(
+                np.tile(255 * colors, (num_pnts, 1)))
+        else:
+            colors = np.asarray(colors)
+            if colors.dtype == object:  # colors is a list of colors
+                color_array = numpy_to_vtk_colors(255 * np.vstack(colors))
+            else:
+                if len(colors) == num_lines:
+                    pnts_colors = np.repeat(colors, pnts_per_line, axis=0)
+                    if colors.ndim == 1:  # Scalar per line
+                        color_array = numpy_support.numpy_to_vtk(pnts_colors,
+                                                                 deep=True)
+                        colors_are_scalars = True
+                    elif colors.ndim == 2:  # RGB(A) color per line
+                        global_opacity = 1 if colors.shape[1] == 3 else -1
+                        color_array = numpy_to_vtk_colors(255 * pnts_colors)
+                elif len(colors) == num_pnts:
+                    if colors.ndim == 1:  # Scalar per point
+                        color_array = numpy_support.numpy_to_vtk(colors,
+                                                                 deep=True)
+                        colors_are_scalars = True
+                    elif colors.ndim == 2:  # RGB(A) color per point
+                        global_opacity = 1 if colors.shape[1] == 3 else -1
+                        color_array = numpy_to_vtk_colors(255 * colors)
+        color_array.SetName('colors')
+        return color_array, colors_are_scalars, global_opacity
+
+    @staticmethod
+    def _points_to_vtk_cells(points):
+        """
+        Returns the VTK cell array for the peaks given the set of points
+        coordinates.
+
+        Parameters
+        ----------
+        points : (N, 3) array or ndarray
+            points coordinates array.
+
+        Returns
+        -------
+        cell_array : vtkCellArray
+            connectivity + offset information.
+
+        """
+        pnts_per_line = 2
```

Comment:
can it be an argument?

Reply:
Added for the 2 functions with that variable.